diff --git a/elpa.pc.in b/elpa.pc.in
index 3b3552c..7ed8639 100644
--- a/elpa.pc.in
+++ b/elpa.pc.in
@@ -8,5 +8,5 @@ Description: ELPA is a Fortran-based high-performance computational library for
 Version: @PACKAGE_VERSION@
 URL: @PACKAGE_URL@
 Libs: -L${libdir} -lelpa@SUFFIX@ @LIBS@ @SCALAPACK_LDFLAGS@ @OPENMP_FCFLAGS@
-Cflags: -I${includedir}/elpa@SUFFIX@-@PACKAGE_VERSION@ @OPENMP_CFLAGS@
+Cflags: -I${includedir}/elpa@SUFFIX@-@PACKAGE_VERSION@ @OPENMP_CFLAGS@ -I${includedir}/elpa@SUFFIX@-@PACKAGE_VERSION@/modules @SCALAPACK_FCFLAGS@ @OPENMP_FCFLAGS@
 fcflags= -I${includedir}/elpa@SUFFIX@-@PACKAGE_VERSION@/modules @SCALAPACK_FCFLAGS@ @OPENMP_FCFLAGS@
